src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -732,6 +737,7 @@ def daemon_type_to_service(dtype: str) -> str:
         'alertmanager': 'alertmanager',
         'prometheus': 'prometheus',
         'node-exporter': 'node-exporter',
+        'ceph-exporter': 'ceph-exporter',
```

Comment:
do we also want this daemon type to be added to the upgrade order? If it's going to be part of th ecpeh image it should definitely be added there.
https://github.com/ceph/ceph/blob/master/src/pybind/mgr/cephadm/utils.py#L24-L30

Reply:
Yes, it'll be a part of the default ceph iage itself. Yeah that cchange I've left for later as currently we're using a image for ceph for testing purpose, but once feature branch is merged in master it'll be the part of ceph image.